File: CSharp/Yow.CoD.Finance.Domain/Contracts/DisburseLoanFundsCommand.cs
using System;

namespace Yow.CoD.Finance.Domain.Contracts
{
    public class DisburseLoanFundsCommand : Command
    {
        public DisburseLoanFundsCommand(Guid commandId, Guid aggregateId, DateTimeOffset transactionDate)  
            : base(commandId, aggregateId)
        {
            TransactionDate = transactionDate;
        }

        public DateTimeOffset TransactionDate { get; }
    }
}

File: java/src/main/java/com/leecampbell/cod/domain/contracts/DisburseLoanFundsCommand.java
package com.leecampbell.cod.domain.contracts;

import java.time.OffsetDateTime;
import java.util.UUID;

public final class DisburseLoanFundsCommand extends DomainCommand {
    private final OffsetDateTime transactionDate;

    public DisburseLoanFundsCommand(UUID commandId, UUID aggregateId, OffsetDateTime transactionDate) {
        super(commandId, aggregateId);
        this.transactionDate = transactionDate;
    }

    public OffsetDateTime getTransactionDate() {
        return this.transactionDate;
    }
}